ingest-app/test/cmr/ingest/test/services/subscriptions_helper.clj

```diff
@@ -94,3 +94,18 @@
             expected (str start "," now)
             actual (#'jobs/subscription->time-constraint data now -1234)]
         (is (= expected actual))))))
+
+(deftest normalize-subscription-search-test
+  "Query normalization, should be sorted parameters"
+  (testing
```

Comment:
nit: CMR coding standard is to put description of the test on the same line of `testing` i.e. `(testing "With a leading question mark"`